analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Invocation.cs

```diff
@@ -33,10 +36,33 @@ internal static class Invocation
                 : context.State;
 
         public static ProgramState Process(SymbolicContext context, IArgumentOperationWrapper argument) =>
-            argument.Parameter is not null      // __arglist is not assigned to a parameter
-            && argument.Parameter.RefKind != Microsoft.CodeAnalysis.RefKind.None
-            && argument.Value.TrackedSymbol() is { } symbol
-                ? context.State.SetSymbolValue(symbol, null)
-                : context.State;
+            ProcessArgument(context.State, argument) ?? context.State;
+
+        private static ProgramState ProcessArgument(ProgramState state, IArgumentOperationWrapper argument)
+        {
+            if (argument.Parameter is null)
+            {
+                return null; // __arglist is not assigned to a parameter
+            }
+            if (argument is { Parameter.RefKind: RefKind.Out or RefKind.Ref, Value: { } value } && value.TrackedSymbol() is { } symbol)
```

Comment:
`TrackedSymbol` doesn't need a null check. So we can remove the `value` and bump coverage
```suggestion
            if (argument is { Parameter.RefKind: RefKind.Out or RefKind.Ref } && argument.Value.TrackedSymbol() is { } symbol)
```